CITATION.cff

```diff
@@ -0,0 +1,27 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+version: 1.8.1
+url: "https://github.com/CQCL/tket"
+preferred-citation:
+  type: article
+  authors:
+  - family-names: "Sivirajah"
+    given-names: "Seyon"
+  - family-names: "Dilkes"
+    given-names: "Silas"
+  - family-names: "Cowtan"
+    given-names: "Alexander"
+  - family-names: "Cowtan"
+    given-names: "Alexander"
+  - family-names: "Simmons"
+    given-names: "Will"
+  - family-names: "Edgington"
+    given-names: "Alec"
+  - family-names: "Duncan"
+    given-names: "Ross"
+  doi: "10.1088/2058-9565/ab8e92"
+  journal: "Quantum Science & Technology"
+  month: 1
```

Comment:
```suggestion
  month: 1
```
I think this should be "number" (if that is the correct tag to use)? The actual month was November.